### DISCUSSION TITLE
Getting errors with nginx proxy_pass on Digital Ocean

### DISCUSSION THREAD

**Discussion by therealJonSnow** [strapi/strapi#6067] [Legacy Topics]
I'm building a nuxt frontend with strapi backend site. Trying to get it to run on a single DO droplet with strapi available through url/strapi/ and the admin at url/strapi/admin. I have got strapi working fine on the url/ and url/admin but cannot get it on this sub root with any success.

The best I've got is the base strapi page showing up on url/strapi/ (with click to open admin link) but when I go to url/strapi/admin it errors with runtime~main.#####.js and main.####.chunk.js 404. I assume this is due to it not looking into the correct folder as these are within build but unsure. 

I am not that experienced with deployment/nginx but I've not managed to find a solution online and its been a week. Any help or advice would be greatly appreciated!

**Comment by derrickmehaffy** [comment]
@therealJonSnow This currently isn't possible with `beta.20.3` but will be possible when `rc.1` is released. On the Strapi beta versions, subfolder based proxying will not work and you have to use a subdomain.

I've already been testing the rc.1 from the mono-repo and have documentation already written and ready for when it's released: https://github.com/strapi/strapi/pull/6134

**Reply by derrickmehaffy** [reply]
For sub-domain help on the beta versions you can refer to this documentation: https://strapi.io/documentation/3.0.0-beta.x/deployment/nginx-proxy.html

**Reply by therealJonSnow** [reply]
Awesome thanks for the response. Got it up and running on sub domain!

**Reply by derrickmehaffy** [reply]
:+1: And be on the look out when the stable is released, if you need to then, I wrote some very detailed guides for sub-folder proxying.

**Comment by hamoudchohan** [comment]
Hi there @derrickmehaffy I have followed the steps from https://strapi.io/documentation/v3.x/deployment/nginx-proxy.html this document and set up the subdomain but I'm facing issues can you please help? I'm new to strapi. 
https://admin.prospiral.co/ this my subdomain.

**Reply by derrickmehaffy** [reply]
Open up a new discussion and add any of your own nginx configs so we can look through them
